blueflood-http/src/main/java/com/rackspacecloud/blueflood/inputs/handlers/HttpMetricsIngestionServer.java

```diff
@@ -62,51 +60,32 @@
 
 public class HttpMetricsIngestionServer {
     private static final Logger log = LoggerFactory.getLogger(HttpMetricsIngestionServer.class);
-    private static TimeValue DEFAULT_TIMEOUT = new TimeValue(5, TimeUnit.SECONDS);
-    private static int WRITE_THREADS = Configuration.getInstance().getIntegerProperty(CoreConfig.METRICS_BATCH_WRITER_THREADS); // metrics will be batched into this many partitions.
-
+    private static TimeValue DEFAULT_TIMEOUT = new TimeValue(1, TimeUnit.MINUTES);
```

Comment:
No, it's just a part of the 202 removal.  I switched the timeoutException handler to now return a 504 instead of a 202; but I thought we agreed to give it more time before returning an error.  How long should we give it?